lib/rspec_logfmt_formatter/formatter.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'rspec/core'
+require 'rspec/core/formatters/base_formatter'
+
+module RspecLogfmtFormatter
+  class Formatter < RSpec::Core::Formatters::BaseFormatter
```

Comment:
I am not a huge fan of this name, however this needs to be in it's own class so that we can have a `version.rb` file. As without it you get a dependancy loop.

Reply:
Looking through a few other RSpec formatters they don't seem to make use of a namespace, it might be nice to try and solve this to be consistent with other projects.

Would it work to mimic the class structure [of this project](https://github.com/CGA1123/rspec_otel_formatter/tree/main/lib), with the class under `lib`, and a `version.rb` in a folder named after the gem that re-opens the class and declares the constant?